fs/operations.go

```diff
@@ -19,6 +21,10 @@ import (
 	"golang.org/x/text/unicode/norm"
 )
 
+// As the output's are printed async, we need a variable to check when the first JSON output has been printed
+var firstJson = false
```

Comment:
I don't like these global variables - I'll explain how to get rid of them below...